### DISCUSSION TITLE
Is it possible to set tbl-colwidths for df-print: paged

### DISCUSSION THREAD

**Discussion by pgseye** [quarto-dev/quarto-cli#7828] [Q&A]
### Description

I am making a Quarto reveals presentation and want to display a couple of data frames in paged view. That works but the columns are quite asymmetric in width and I'd like to make all 4 of them equal.

I've tried the following but it doesn't do anything. Is it possible to change printed df col widths when setting df-print:paged?

```{r rows.print = 20}
#| tbl-colwidths: [25,25,25,25]
df
```

**Comment by mcanouil** [comment]
As described in the documentation, this option is mostly for markdown tables, see <https://quarto.org/docs/reference/formats/html.html#tables>.
"paged" is HTML/JavaScript table.

If you want more control, then you have to use dedicated R packages and handle the size within those R packages such as `knitr` with `kable()` and `kableExtra`, `gt`, `flextable`, `DT`, etc.

**Reply by pgseye** [reply]
ok thanks. I think I have found that `kableExtra` can do some of what I want.
